data/configuration.yml

```diff
@@ -65,8 +65,14 @@ blacklist:
     path: https://raw.githubusercontent.com/dnswarden/blocklist/master/blacklist-formats/hosts
   - format: domains
     path: https://raw.githubusercontent.com/nextdns/metadata/master/parentalcontrol/bypass-methods
+  - format: hosts
+    path: https://raw.githubusercontent.com/ejrv/VPNs/master/vpn-ipv4.txt
+  - format: hosts
+    path: https://raw.githubusercontent.com/ejrv/VPNs/master/vpn-ipv6.txt
```

Comment:
Unfortunately, this won't work. We can only block actual domain names using our adblock DNS. If user uses ip address directly, it will not go through our DNS

Reply:
I think just remove these entries.